test/configCases/commons-chunk-plugin/move-to-grandparent/index.js

```
@@ -0,0 +1,12 @@
+it("should correctly include indirect children in common chunk", function(done) {
+	Promise.all([
+		import('./pageA'),
+		import('./pageB')
+	]).then((imports) => {
+		imports[0].default.should.be.eql("reuse");
+		imports[1].default.should.be.eql("reuse");
+		done();
+	}).catch(e => {
+		done();
```

Pass the error to the done callback, elsewise errors are ignored.